README.md

```diff
@@ -0,0 +1,9 @@
+# Java Core June 2021
```

Comment:
Please merge with master

Reply:
I do not under understand what i have to do with this point... Pull is saing that everything is up-to-date